python/tvm/topi/cuda/sparse.py

```diff
@@ -105,13 +105,22 @@ def _callback(op):
     return s
 
 
-def schedule_cuda_transpose(s, out):
+def schedule_transpose(outs):
```

Comment:
feels a bit weird to have this in `sparse.py`

Reply:
moved to transform.py